platform/time/bundle.json

```diff
@@ -0,0 +1,19 @@
+{
+    "name": "Time services bundle",
+    "description": "Defines interfaces and provides default implementations for handling different time systems.",
+    "extensions": {
+        "components": [
+            {
+                "provides": "timeService",
+                "type": "aggregator",
+                "implementation": "TimeAggregator.js"
+            },
+            {
+                "provides": "timeService",
```

Comment:
Seems like the composite service pattern is overkill when we're really looking for a type registry, and I'm concerned it is more confusing when we implement aggregators that expose a different interface than the interface of the individual providers.  (I have been known to do this... say in the search aggregator).

What do you think about using a new extension category for time formats instead of composite services?